backend/ormconfig.json

```diff
@@ -10,6 +10,7 @@
   "logging": false,
   "entities": ["src/main/entity/**/*.ts"],
   "migrations": ["src/main/migration/**/*.ts"],
+  "migrationsRun": true,
```

Comment:
Basically, this will run any migrations inside `src/main/migration`.
https://typeorm.io/#/migrations